contracts/contracts/NoLossDao_v0.sol

```diff
@@ -39,7 +39,7 @@ contract NoLossDao_v0 is Initializable {
   mapping(address => address) public voteDelegations; // For vote proxy
 
   //////// Necessary to fund dev and miners //////////
-  address[] interestReceivers;
+  address[] interestReceivers; // in v0, the interestReceivers is the address of the miner.
```

Comment:
Well, interestReceivers[0] is the miners address

Reply:
That's what I meant 😬